qiskit_experiments/library/randomized_benchmarking/mirror_rb_analysis.py

```diff
@@ -0,0 +1,674 @@
+
+    # section: fit_parameters
+        defpar a:
+            desc: Height of decay curve.
+            init_guess: Determined by :math:`1 - b`.
+            bounds: [0, 1]
+        defpar b:
+            desc: Base line.
+            init_guess: Determined by :math:`(1/2)^n` (for success probability) or :math:`(1/4)^n`
+            (for adjusted success probability and effective polarization).
+            bounds: [0, 1]
+        defpar \alpha:
+            desc: Depolarizing parameter.
+            init_guess: Determined by :func:`~rb_decay` with standard RB curve.
+            bounds: [0, 1]
+
+    # section: reference
+        .. ref_arxiv:: 1 2112.09853
+
+    """
+
+    def __init__(self):
+        super().__init__(
+            models=[lmfit.models.ExpressionModel(expr="a * alpha ** x + b", name="mirror")]
+        )
+        self._gate_counts_per_clifford = None
+        self._physical_qubits = None
+        self._num_qubits = None
+
+    @classmethod
+    def _default_options(cls):
+        """Default analysis options.
+
+        Analysis Options:
+            y_axis (str): Set the metric to plot on the y-axis. Must be one of
+                "Effective Polarization" (default), "Success Probability", or "Adjusted
+                Success Probability".
+            gate_error_ratio (Optional[Dict[str, float]]): A dictionary with gate name keys
+                and error ratio values used when calculating EPG from the estimated EPC.
+                The default value will use standard gate error ratios.
+                If you don't know accurate error ratio between your basis gates,
+                you can skip analysis of EPGs by setting this options to ``None``.
+            epg_1_qubit (List[AnalysisResult]): Analysis results from previous RB experiments
+                for individual single qubit gates. If this is provided, EPC of
+                2Q RB is corrected to exclude the depolarization of underlying 1Q channels.
+        """
+        default_options = super()._default_options()
+
+        # Set labels of axes
+        default_options.plotter.set_figure_options(
+            xlabel="Clifford Length",
+            ylabel="Effective Polarization",
+        )
+
+        # Plot all (adjusted) success probabilities
+        default_options.plot_raw_data = True
+
+        # Exponential decay parameter
+        default_options.result_parameters = ["alpha"]
+
+        # Default gate error ratio for calculating EPG
+        default_options.gate_error_ratio = "default"
+
+        # By default, EPG for single qubits aren't set
+        default_options.epg_1_qubit = None
+
+        # By default, effective polarization is plotted (see arXiv:2112.09853). We can
+        # also plot success probability or adjusted success probability (see PyGSTi).
+        # Do this by setting options to "Success Probability" or "Adjusted Success Probability"
+        default_options.y_axis = "Effective Polarization"
```

Comment:
`y_axis` sounds bit strange to me. Y axis is a configuration of visualization, neither more nor less. This can be something like analyzed_quantity or feature_value? I think you have better wording in this context (@eggerdj should have a nice word lol).

Reply:
Good point, changed to `analyzed_quantity`.